### PR TITLE
AusPIX removed from normative standing

### PR DESCRIPTION
**Pull request by nicholascar**
As discussed two meeting back, here AusPIX DGGS is removed from normative standing but retained as an example for DGGSes.

### CODE REVIEW

File: 1.1/spec/11-Part-08.adoc [hunk 1]
```diff
@@ -615,12 +605,12 @@ geo:asDGGS a rdf:Property, owl:DatatypeProperty ;
...
     rdfs:isDefinedBy geo: ;
     skos:prefLabel "as DGGS"@en ;
-    skos:definition "A DGGS Well-Known Text serialization of a geometry."@en ;
+    skos:definition "A DGGS serialization of a geometry."@en ;
     rdfs:domain geo:Geometry ;
     rdfs:range geo:dggsLiteral .
 ```
 
-NOTE: It is expected that this property will be used to indicate specific DGGS data types, such as http://www.opengis.net/ont/geosparql#auspixDggsLiteral[`geo:auspixDggsLiteral`], descibed above, as opposed to the generic http://www.opengis.net/ont/geosparql#dggsLiteral[`geo:dggsLiteral`].
+NOTE: It is expected that this property will be used to indicate specific DGGS data types, such as the example `eg:auspixDggsLiteral`, descibed above, as opposed to the generic http://www.opengis.net/ont/geosparql#dggsLiteral[`geo:dggsLiteral`].
```
Comment (FransKnibbe) [inline 1.1/spec/11-Part-08.adoc:613]: typo: descibed

File: 1.1/spec/11-Part-08.adoc [hunk 2]
```diff
@@ -548,10 +548,10 @@ The function http://www.opengis.net/def/function/geosparql/asKML[`geof:asKML`] c
...
+Here is defined one RDFS Datatypes:
+http://www.opengis.net/ont/geosparql#dggsLiteral[`+http://www.opengis.net/ont/geosparql#dggsLiteral+`] and one property, http://www.opengis.net/ont/geosparql#asDGGS[`+http://www.opengis.net/ont/geosparql#asDGGS+`]. 
 
-NOTE: The two datatypes defined here are for an abstract DGGS implementation (http://www.opengis.net/ont/geosparql#dggsLiteral[`geo:dggsLiteral`]) and a specific one, AusPIX <<AUSPIX>> (http://www.opengis.net/ont/geosparql#auspixDggsLiteral[`geo:auspixDggsLiteral`]). The purposes of including both of these are to indicate the conceptual position of DGGS literals in this specification - the abstract form acts as a parent of all other potential, specific, DGGS datatypes - and to exemplify a specific system implementation which allows for real examples of use.
+NOTE: The datatype defined here is for an abstract DGGS implementation (http://www.opengis.net/ont/geosparql#dggsLiteral[`geo:dggsLiteral`]) but concrete ones should be used in real implementations. For example, the AusPIX DGGS <<AUSPIX>> might implement something similar to `eg:auspixDggsLiteral`.
```
Comment (FransKnibbe) [inline 1.1/spec/11-Part-08.adoc:554]: Isn't it more common (and therefore more understandable) to use a prefix "ex" for example IRIs that are not meant to be resolved?

**Review by FransKnibbe — APPROVED** [review]

File: 1.1/spec/11-Part-08.adoc (hunk 2 above)
Comment (nicholascar) [reply 1.1/spec/11-Part-08.adoc]: I've changed to ex:

File: 1.1/spec/11-Part-08.adoc (hunk 1 above)
Comment (nicholascar) [reply 1.1/spec/11-Part-08.adoc]: Fixed

**Pull request merged**